### DISCUSSION TITLE
Short Key of fast forward / fast rewind of video preview （视频预览是否支持用快捷键进行快进、快退）

### DISCUSSION THREAD

**Discussion by arroy2024** [QL-Win/QuickLook#1460] [General]
Is there any short keys for fast forward / fast rewind of video preview？Thanks.

求助，预览视频文件时候，有没有快捷键进行快进和快退？另外如果有人在用Quciker，可否提示下如何实现，我刚刚开始学习。

**Comment by emako** [comment]
Since 3a1a74d, the keyboard listening can be implemented from WPF events

**Reply by arroy2024** [reply]
> Since [3a1a74d](https://github.com/QL-Win/QuickLook/commit/3a1a74d0f94ddd312ba02187096ea03ac59e72af), the keyboard listening can be implemented from WPF events

Sorry I'm not a developer, and I am not familiar with WPF. Dou you mean I need to set something to achieve that?

**Comment by emako** [comment]
Support in future version
